lib/measures/tbd/README.md

```diff
@@ -43,6 +43,14 @@ Write tbd.out.json to customize for subsequent runs. Edit and place in model fil
 **Required:** true,
 **Model Dependent:** false
 
+### Generate Kiva inputs
+Generate OSM Kiva settings and objects if model surfaces have foundation boundary conditions
```

Comment:
I'd suggest the behavior be that ground surfaces are converted to foundation surfaces rather than asking the user change surfaces to foundation beforehand.

Reply:
Still needs extensive testing, but the --force Kiva checkbox/argument should do the trick.